views/partials/pasteSettings.ejs

```diff
@@ -28,37 +28,12 @@
     <p id="previewURL">(https://www.imperialb.in/p/documentID)</p>
     <p>How long do you want your pastes to stay for? (days)</p>
     <select name="Day" id="day">
-      <option value="1">1</option>
-      <option value="2">2</option>
-      <option value="3">3</option>
-      <option value="4">4</option>
-      <option value="5">5</option>
-      <option value="6">6</option>
-      <option value="7" selected>7</option>
-      <option value="8">8</option>
-      <option value="9">9</option>
-      <option value="10">10</option>
-      <option value="11">11</option>
-      <option value="12">12</option>
-      <option value="13">13</option>
-      <option value="14">14</option>
-      <option value="15">15</option>
-      <option value="16">16</option>
-      <option value="17">17</option>
-      <option value="18">18</option>
-      <option value="19">19</option>
-      <option value="20">20</option>
-      <option value="21">21</option>
-      <option value="22">22</option>
-      <option value="23">23</option>
-      <option value="24">24</option>
-      <option value="25">25</option>
-      <option value="26">26</option>
-      <option value="27">27</option>
-      <option value="28">28</option>
-      <option value="29">29</option>
-      <option value="30">30</option>
-      <option value="31">31</option>
+      <script>
+        // Quick way to add 31 options.
+        for(let i = 1; i <= 31; i++) {
+          document.getElementById("day").innerHTML += `<option value=${i}>${i}</option>`;
+        }
```

Comment:
You can omit the loop and stop querying the element every single recursion by using the following:

```ts
document.getElementById("day").innerHTML = [...new Array(31)].map((_, i) => `<option value="${i + 1}">${i + 1}</option>`);
```

Reply:
damn dude pulled out moonrunes wtf

Reply:
Sorry, this needs a `.join('')` on the end to concatenate all the values into a string.

Becomes
```ts
document.getElementById("day").innerHTML = [...new Array(31)].map((_, i) => `<option value="${i + 1}">${i + 1}</option>`).join('');
```

Reply:
```
for(let i = 1; i <= 31; i++) document.getElementById("day").innerHTML += `<option value=${i}>${i}</option>`;
```        
I can pull off a 1 liner as well.

Reply:
> You can omit the loop and stop querying the element every single recursion by using the following:
> 
> ```ts
> document.getElementById("day").innerHTML = [...new Array(31)].map((_, i) => `<option value="${i + 1}">${i + 1}</option>`);
> ```

I'm pretty sure this is harder to understand than just having a for loop.

Reply:
Your code is querying the element every single time. One liner doesn't mean increased readability, but its better to map through. If cleanliness is the primary issue with the one liner, then this is the best simplification.

```ts
const options = [...new Array(31)].map((_, i) => {
  return `<option value="${i + 1}">${i + 1}</option>`;
});

document.getElementById("day").innerHTML = options.join("\n");
```

Reply:
Not to mention, there are two missing `"` surrounding the value attribute, so the HTML spat out would not be valid.